Gemfile

```diff
@@ -26,20 +26,38 @@ gem 'puma', '~> 3.11'
 gem 'bootsnap', '>= 1.1.0', require: false
 
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
-# gem 'rack-cors'
+gem 'rack-cors'
 
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'rspec-rails', '~> 3.5'
 end
 
 group :development do
   gem 'listen', '>= 3.0.5', '< 3.2'
+  # Ruby linter
+  gem 'rubocop', '~> 0.56.0', require: false
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  # Preview email in browser
+  gem "letter_opener", "~> 1.6"
+  gem 'annotate'
```

Comment:
When using a new gem, go to rubygems.com and copy from there to get the last version so you add the gem with a specific version (i.e the one above, letter_opener)